src/frontend/components/Loading/Loading.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
```

Comment:
As I said before, I am not a big fan os this approach. If something fails here , you will struggle to debug it due to its complexity.

Its ok for now, but please avoid this kind of copy,paste approaches in the long term!

Reply:
Ok. Thank you review and advice.